docs/examples/java-agent-injector-usage.md

```diff
@@ -34,7 +34,7 @@ spec:
     spec:
       containers:
       - name: demo1
-        image: dashanji/swck-spring-demo:v0.0.3
+        image: ghcr.io/apache/skywalking-swck-spring-demo:v0.0.1
```

Comment:
@dashanji I renamed this demo image and push it to GitHub container repository.

BTW, What paths we could access in this app? I got a `404` on the root path.

Reply:
Sorry for not giving instructions，we could use `curl http://localhost:(port-forward IP)/hello` to access it. Next , I will add an e2e test based on `skywalking-infra-e2e`, i think it's more convenient to test.